plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -196,16 +201,17 @@ type VolumeFilter struct {
 // the maximum.
 func NewMaxPDVolumeCountPredicate(filter VolumeFilter, maxVolumes int, pvInfo PersistentVolumeInfo, pvcInfo PersistentVolumeClaimInfo) algorithm.FitPredicate {
 	c := &MaxPDVolumeCountChecker{
-		filter:     filter,
-		maxVolumes: maxVolumes,
-		pvInfo:     pvInfo,
-		pvcInfo:    pvcInfo,
+		filter:               filter,
+		maxVolumes:           maxVolumes,
+		pvInfo:               pvInfo,
+		pvcInfo:              pvcInfo,
+		randomVolumeIDPrefix: rand.String(8),
```

Comment:
Can this be 32 like originally?  That way we reduce the chance of conflicting with the user's object names.

Reply:
Done.